fe/fe-core/src/main/java/com/starrocks/memory/MemoryTrackable.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.memory;
+
+import org.apache.spark.util.SizeEstimator;
+
+public interface MemoryTrackable {
+
+    default long estimateSize() {
+        return SizeEstimator.estimate(this);
+    }
+
+    long estimateCount();
```

Comment:
Your suggestion is very good, but I think you should not add it for now. You can add it later, because even if it is added now, this function cannot be found. When we record the log, we also record the overall value and do not record it too finely.